sdks/android/Makefile

```diff
@@ -8,16 +8,19 @@ NDK_BUILD = $(NDK_DIR)/ndk-build
 PACKAGE   = org.mono.android.AndroidTestRunner
 ACTIVITY  = org.mono.android.AndroidRunner
 
+PROJECT_SETUP_FILES = project.properties local.properties
+
+
 .PHONY: all
 all: setup stage-sdk
 	$(MAKE) -C managed all
 	$(NDK_BUILD)
 	$(ANT) debug
 
-local.properties:
+$(PROJECT_SETUP_FILES): build.xml AndroidManifest.xml
```

Comment:
I am slightly changing how that is done in https://github.com/mono/mono/pull/5894